web/src/AlertPane.scss

```diff
@@ -66,6 +66,7 @@
 
 // ...only show last few lines
```

Comment:
FWIW, this CSS for hiding the lines can be more simple and clear:

```
// Hide all lines by default
.AlertPane-item code {
  white-space: pre-wrap;
  display: none;
}
```

I had `height: 0;` and `opacity: 0;` previously because I was getting ready for a "show more" button that would reveal all the lines with smooth animation. Which I never ended up actually implementing. 

(When you use `display: none` and `display: [something]` for showing and hiding, it takes away your animation options.)

Reply:
much nicer!